tools/gfx/dummy-render-pass-layout.h

```diff
@@ -0,0 +1,30 @@
+// dummy-render-pass-layout.h
+#pragma once
+
+// Implementation of a dummy render pass layout object that stores and holds its
+// desc value. Used by targets that does not expose an API object for the render pass
+// concept.
+
+#include "slang-gfx.h"
+#include "slang-com-helper.h"
+#include "core/slang-basic.h"
+
+namespace gfx
+{
+
+class DummyRenderPassLayout
```

Comment:
Naming nit: the use of "dummy" as a name in code is something people could find offensive even if we don't intend it. It might be better to have this be called a "placeholder," "proxy," or "simple" render pass layout.

Reply:
Renamed to "Simple".